src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -215,28 +194,56 @@ export class MeetingManager implements AudioVideoObserver {
   }
 
   async join({
-    meetingInfo,
-    attendeeInfo,
-    deviceLabels = DeviceLabels.AudioAndVideo,
+    meetingSessionConfiguration,
+    deviceLabels,
     eventReporter,
     meetingManagerConfig,
```

Comment:
Ask again whether we need meetingManagerConfig :)

Reply:
So `meetingManagerConfig` is config for the `MeetingManager` when constructed, it will also be useful to get the `logger` like objects which can be used across the `MeetingProvider` mount-un-mount lifecycle. Hence, should be ok to keep. Otherwise, anyhow we have to have props to receive `DeviceLabels`, `enableWebAudio` or `PostLogger` config.

Reply:
for me it is confusing to have `MeetingJoinData` which then have to contains `MeetingManagerConfig` which in turn confusing with `MeetingSessionConfiguration`. From the caller perpsective I could not tell the difference between all of these :D

Reply:
Also `DeviceLabels` is already outside of `MeetingManagerConfig`.

Reply:
I prefer second method. In my opinion, `join` and `configure` should be two steps but currently we are mixing them.

The `Props` of `MeetingProvider` and `MeetingManagerConfig` is confusing me. Especially when you want to merge them or tell the difference between them. This indicates that it is a good design. In my opinion I think it would be great to remove the properties of `MeetingManagerConfig` from `MeetingProvider` constructor. And follow the second option.

Reply:
All the properties in `meetingManagerConfig` are for JS SDK so it is not really a `config` object so first option for me. We should no longer have any props in meetingProvider either.

Reply:
First option works for me as well. I just want to remove the `MeetingManagerConfig` from `MeetingProvider`.

Reply:
Implemented the first option